### PR TITLE
Update-Set-MpPreference-4519186

### PR DESCRIPTION
**Pull request by Lovina-Saldanha**
Updated information for the parameter "DisableCpuThrottleOnIdleScans"

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

**Review by tseward — APPROVED** [review]

File: docset/windows/defender/Set-MpPreference.md [hunk 1]
```diff
@@ -245,6 +245,18 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DisableCpuThrottleOnIdleScans
+Indicates whether the CPU will be throttled for scheduled scans while the device is idle. This parameter is enabled by default, thus ensuring that the CPU will not throttle for scheduled scans performed when the device is idle, regardless of what **ScanAvgCPULoadFactor** is set to. For all other scheduled scans, this flag does not have any impact and normal throttling will occur.
```
Comment (JohanFreelancer9) [inline docset/windows/defender/Set-MpPreference.md:249]:
```suggestion
Indicates whether the CPU will be throttled for scheduled scans while the device is idle. This parameter is enabled by default, thus ensuring that the CPU will not be throttled for scheduled scans performed when the device is idle, regardless of what **ScanAvgCPULoadFactor** is set to. For all other scheduled scans, this flag does not have any impact and normal throttling will occur.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by get-itips — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Pull request merged**